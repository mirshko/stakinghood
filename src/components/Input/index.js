import styled from "styled-components";

const Input = styled.input`
  width: 100%;
  margin: 24px 0;
  padding: 12px 16px;
  border-radius: 4px;
  border: none;
  line-height: 1.5;
  font-size: 16px;
  background-color: var(--brand-light);
  color: #000;

  ::placeholder {
    color: #000;
  }
`;

export default Input;
