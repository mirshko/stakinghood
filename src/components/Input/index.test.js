import React from "react";
import { shallow } from "enzyme";

import Input from "./index";

describe("<Input />", () => {
  it("it works", () => {
    const wrapper = shallow(<Input />);
    expect(wrapper).toMatchSnapshot();
  });
});
