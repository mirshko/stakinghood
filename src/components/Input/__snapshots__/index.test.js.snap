// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  width: 100%;
  margin: 24px 0;
  padding: 12px 16px;
  border-radius: 4px;
  border: none;
  line-height: 1.5;
  font-size: 16px;
  background-color: var(--brand-light);
  color: #000;

  ::placeholder {
    color: #000;
  }
",
        ],
      },
      "displayName": "styled.input",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "input",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
